docs/source/styling.rst

```diff
@@ -117,6 +121,10 @@ Some block tags may not work on old versions.
     {% block submit_button_wrapper_cls %}col-sm-3 col-md-2 px-2 m-3 m-sm-0{% endblock submit_button_wrapper_cls %}
     {% block submit_button_cls %}btn btn-outline-success btn-block btn-sm{% endblock submit_button_cls %}
 
+    {% block login_info %}
+    {# Please see 'comment/comments/create_comment.html' template for default block #}
```

Comment:
Maybe we can place a symlink here.

Reply:
I think hyperlink is not supported inside code block.